packages/rancher-gatekeeper/overlay/CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# Changelog
+All notable changes from the upstream OPA Gatekeeper chart will be added to this file
+
+## [Package Version 00] - 2020-07-27
+### Added
+- Added `validate_crds_installed.yaml` to validate crds are installed before installing racher-gatekeeper
+- Added the following crd annotations for `rancher-gatekeeper-crd` chart dependency:
+    - `catalog.cattle.io/requires-gvr: configs.config.gatekeeper.sh/v1alpha1`
+    - `catalog.cattle.io/auto-install-gvr: configs.config.gatekeeper.sh/v1alpha1`
```

Comment:
@cbron What commits would you like to see squashed? I initially planned doing less commits, but ended up doing it this way because reviewing a patch file with a lot of changes can be pretty awful at times. As for using the CRD generator I agree, but it really depends on how soon we want this merged so I'll leave that up to @deniseschannon.